sql/src/main/java/io/crate/metadata/IndexParts.java

```diff
@@ -43,6 +44,8 @@
     @VisibleForTesting
     public static final String PARTITIONED_TABLE_PART = "." + PARTITIONED_KEY_WORD + ".";
 
+    public static final List<String> DANGLING_INDICES_PREFIX_PATTERNS = ImmutableList.of(".shrinked.*");
```

Comment:
should we make this more generic to account for any dot-prefixed index (apart from blob and partitions)?

Reply:
I'm reluctant to be overly eager with deleting unspecified indices.

Reply:
To elaborate: Currently there is no other way to create `.whatever` indices using our exposed API. So the choice here is mainly about future proofing where you have two kind of bugs if we forget to adapt things: We delete too much or too little. I'd go with the latter.

Reply:
Not sure what that has to do wether we log the indices at startup or not. What are your concerns here?